enterprise/admin/forms.py

```diff
@@ -0,0 +1,75 @@
+# -*- coding: utf-8 -*-
+"""
+Custom admin forms.
+"""
+from __future__ import absolute_import, unicode_literals
+
+from django import forms
+from django.contrib.auth.models import User
+from django.core.exceptions import ValidationError
+from django.core.validators import validate_email
+from django.utils.translation import ugettext as _
+
+from enterprise import utils
+from enterprise.models import EnterpriseCustomer, EnterpriseCustomerUser
+
+
+class ManageLearnersForm(forms.Form):
+    """
+    Form to manage learner additions.
+    """
+    email = forms.CharField(label=_("Email or Username"))
+
+    def clean_email(self):
+        """
+        Clean email form field
+
+        Returns:
+            str: email to link
+        """
+        message_template = _(
```

Comment:
Any reason to define this here instead of where it's used below (it's only used once)?